khal/khalendar/event.py

```diff
@@ -524,8 +524,9 @@ def categories(self) -> str:
 
     def update_categories(self, categories: List[str]) -> None:
         assert isinstance(categories, list)
+        categories = [c.strip().lower() for c in categories if c != ""]
```

Comment:
`.lower()` is not correct here; categories may be uppercase or mixed case.

Reply:
That's right. I removed it.